stock_valued_picking_report/report/stock_picking_valued_report.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="valued_report_picking" inherit_id="stock.report_delivery_document">
```

Comment:
If report is stock.report_delivery_document instead of stock.report_picking, taxes are printed in delivery slip report instead of pickings operations report.

Either you should change this or change README file.